pkg/utils/helm/helm_helper.go

```diff
@@ -295,6 +329,9 @@ func (h *Helper) GetValuesFromChart(repoURL string, chartName string, version st
 		if err != nil {
 			continue
 		}
+		if h.cache != nil {
+			h.cache.Put(fmt.Sprintf(valuesPatten, repoURL, chartName, version), c.Values, 10*time.Minute)
```

Comment:
Dynamically calculate cache time same with index info?

Reply:
this line is store values in cache. index cache is already dynamically calculated.